onfido-java/src/main/java/com/onfido/Onfido.java

```diff
@@ -63,7 +63,7 @@ private Onfido(Builder builder) {
     /** The Api token. */
     public String apiToken = "";
     /** The Api url. */
-    public String apiUrl = DEFAULT_API_URL;
+    public String apiUrl = EU_API_URL;
```

Comment:
@ciaran16 @JakeBat in hindsight, if we don't want a default, presumably this should be `"";`?

Reply:
Yeah, that would make sense, if we leave it this way, my new check would be pointless, unless Ciaran objects I would say update it.